package.json

```diff
@@ -22,7 +22,7 @@
     "doc": "run-s doc:html && open-cli build/docs/index.html",
     "doc:html": "typedoc --out build/docs",
     "doc:json": "typedoc --json build/docs/typedoc.json",
-    "docs:generate:markdown": "typedoc --options typedoc.json --out testdocs",
+    "docs:generate:markdown": "typedoc --options typedoc.json --out ./docs/techincal-reference --readme none",
```

Comment:
I suggest to add the out in the `typedoc.json`, so it is consistent with: 
https://github.com/ERC725Alliance/erc725.js/blob/develop/typedoc.json

Reply:
There is already an out option in this config file, let's not define A in config file and overwrite to B in cli.